addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -1041,12 +1043,48 @@
 		
 		"406"	//Splendid Screen
 		{
-			"prefab"		"131"
+			"desp"			"Splendid Screen: {positive}On bash: increased damage and knockback"
+			"attrib"		"791 ; 1.5 ; 248 ; 3.0"
+			
+			"attackdamage"
+			{
+				"params"
+				{
+					"damagetype"	"knockback"
+				}
+				
+				"Tags_AddAttrib"
+				{
+					"target"		"victim"
+					"cond"			"127"
+					"duration"		"0.5"
```

Comment:
wasnt airblast cond suppose to be perma until touches ground, or was that too much

Reply:
nah i just forgot to update that on my config loool

Reply:
Should add comment to note duration is perma until touches ground